app/views/templates/script/convert2rhel_analyze.erb

```diff
@@ -0,0 +1,49 @@
+<%#
+name: Convert2RHEL analyze
+snippet: false
+template_inputs:
+  - name: Data telemetry
+    required: false
+    input_type: user
+    options: "yes\r\nno"
+    advanced: false
+    value_type: plain
+    default: 'yes'
+    hidden_value: false
+model: JobTemplate
+job_category: Convert 2 RHEL
+provider_type: script
+kind: job_template
+%>
+<% if @host.operatingsystem.family == 'Redhat' -%>
+if ! [ $(id -u) = 0 ]; then
```

Comment:
`=` in test does string-based comparison. I'd rather be explicit and use `-eq` to do numeric comparison